sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionSuite.scala

```diff
@@ -337,6 +337,22 @@ class StatisticsCollectionSuite extends StatisticsCollectionTestBase with Shared
     }
   }
 
+  test("auto gather stats after insert command") {
+    val table = "change_stats_insert_datasource_table"
+    val autoUpdate = true
+      withSQLConf(SQLConf.AUTO_SIZE_UPDATE_ENABLED.key -> autoUpdate.toString) {
```

Comment:
Thank you for updating, @sujith71955 . Here are two issues.
1. We need to test both cases by using `Seq(false, true).foreach { autoUpdate =>`
2. The current indentation is wrong at 343.

If you fix (1), (2) will be fixed together.

Reply:
Handled the comment. thanks